dockerfiles/Dockerfile

```diff
@@ -23,8 +23,7 @@ ARG VERSION
 RUN set -e ;\
     cd /opt/labgrid ;\
     pip3 install yq ;\
-    pip3 install --no-cache-dir -r requirements.txt ;\
-    SETUPTOOLS_SCM_PRETEND_VERSION="$VERSION" python3 setup.py install ;\
+    SETUPTOOLS_SCM_PRETEND_VERSION="$VERSION" pip3 install --no-cache-dir . ;\
```

Comment:
Why is `--no-cache-dir .` needed?

Reply:
No idea, it was there before, so I left it there.

Reply:
Presumably because it avoids taking up unnecessary space, thus keeping the docker image that little bit smaller.